main/guides/getting-started/sell-concert-tickets-contract-explainer.md

```diff
@@ -15,17 +22,39 @@ const inventory = {
   middleRow: {
     tradePrice: AmountMath.make(istBrand, 2n),
     maxTickets: 5n,
+  },
+  backRow: {
+    tradePrice: AmountMath.make(istBrand, n),
+    maxTickets: n,
   }
 };
 ```
+Our contract uses this `inventory` object as a parameter to initiate. 
```

Comment:
*initiate* needs an object. Maybe change it to *Our contract takes the provided inventory object as a parameter to initiate the process*

Reply:
Fixed it.